Src/CSharpier/SyntaxPrinter/ArgumentListLikeSyntax.cs

```diff
@@ -10,14 +10,15 @@ FormattingContext context
     )
     {
         var docs = new List<Doc> { Token.Print(openParenToken, context) };
+        var lambdaId = Guid.NewGuid();
 
         switch (arguments)
```

Comment:
I forgot to mention this in your original PR, but this all branches of this switch statement just add something to the list.
Wouldn't it look better if you used the switch expression instead?

Reply:
The expression looks almost the same for me